src/components/product-menu-dropdown/product-menu-dropdown.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import Icon from '@mapbox/react-icon';
+
+class ProductMenuDropdown extends React.PureComponent {
+  render() {
+    const categoryLength = this.props.categories.length;
+    const activeDotStyle = {
+      content: '',
+      width: '6px',
+      height: '6px',
+      borderRadius: '3px',
+      background: '#4264fb',
+      position: 'absolute',
+      margin: '10px 0 0 -12px',
+      display: 'none'
+    };
+    const allCategories = this.props.categories.map((category, index) => {
+      const allProducts = category.products.map((product, index) => {
+        const liClasses = classnames('mt6 relative ', {
+          'txt-bold': product.url && location.pathname.indexOf(product.url) > -1
+        });
+        const dotClasses = classnames({
+          'inline-block': location.pathname.indexOf(product.url) > -1,
```

Comment:
You could consolidate this logic into a single variable: `isActive`. Then you only have to figure it out once and can reapply it in these various contexts.